src/webparts/LaserficheRepositoryAccessWebPart/components/LaserficheRepositoryAccessWebPart.tsx

```diff
@@ -220,6 +219,7 @@ export default class LaserficheRepositoryAccessWebPart extends React.Component<
       showUploadModal: false,
       showCreateModal: false,
       showAlertModal: false,
+      region: this.props.devMode ? "a.clouddev.laserfiche.com" : "laserfiche.com",
```

Comment:
how about cloudtest?

Reply:
For now, it's just clouddev/production, we can change this to a dropdown and enum later if needed